packages/@o3r/extractors/package.json

```diff
@@ -65,6 +64,7 @@
     "rxjs": "^7.4.0",
     "ts-jest": "^28.0.5",
     "type-fest": "^2.13.1",
+    "typedoc": "~0.23.5",
```

Comment:
This is incorrect, it is actually a real `depedency`.
It is used with `import type` in the code but it is part of exposed types (via the `public_api.ts` file) so the user will need the dependency to resolve the typings, it is not used only for build.
(It would be fine only if the types would not have been exposed, such as schematics usage)